app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -1,10 +1,39 @@
 import React from "react";
 
+import * as yup from "yup";
+
 import { convertDateToStdFormat } from "utils/dayjs 4.31.02 PM";
 
 import Avatar from "./Table/Avatar";
 import RowActions from "./Table/RowActions";
 
+// Start of Form component constants. --------------------------------
+export const CONTACT_FORM_INITIAL_FORM_VALUES = {
+  firstName: "",
+  lastName: "",
+  email: "",
+  role: "",
```

Comment:
In your validation you have mentioned `role` is an object. But here you have set it as an empty string. It doesn't make sense to my. Fix the disparity.